tools/firtool/firtool.cpp

```diff
@@ -228,6 +228,17 @@ static cl::opt<bool>
                 cl::desc("run the reset inference pass on firrtl"),
                 cl::init(true), cl::cat(mainCategory));
 
+static cl::opt<bool> exportChiselInterface(
```

Comment:
This seems appropriate to me, although looks like this will output to stdout by default which is also the default output for firtool, causing both on stdout (not interleaved, but still).

Reply:
I could remove the `exportChiselInterface` option and have `chiselInterfaceOutDirectory` only, maybe renamed to something like `chiselInterfaceExportDirectory`. If someone wanted to directly output a Chisel interface to stdout they could still use `circt-opt ... --export-chisel-interface`.